src/styles/calendar.scss

```diff
@@ -0,0 +1,41 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: maxWidth($squareWidth, $padding);
+  gap: $gap;
+  padding: $padding;
+
+  &__day {
+    width: $squareWidth;
+    height: $squareWidth;
+    background-color: #eee;
+    border: 1px solid black;
+    position: relative;
+  }
+
+  @for $day from 1 through 31 {
+    &__day--#{$day}::before {
+      content: "#{$day}";
+      font-size: 30px;
+      font-family: Arial, Helvetica, sans-serif;
+      position: absolute;
+      left: 50%;
+      transform: translateX(-50%);
+      line-height: $squareWidth;
```

Comment:
it's important not to add repeated rules within the cycle. Doing so would result in unnecessary code duplication, which can lead to:
1. Increased file size: Repeated rules will cause the generated CSS file to be larger than necessary, leading to longer loading times for your website.
2. Difficulty in maintenance: Duplicate code can make it harder to maintain and update your styles, as changes may need to be made in multiple places.
3. Decreased readability: More code means it's harder for you or others to understand the structure and purpose of your SASS file.
By keeping your SASS code clean and free of repeated rules, you'll ensure that your generated CSS is efficient, maintainable, and easy to understand.

 `content: "#{$day}";` <= changes
 `font-size: 30px;` <= static